content/teams/teams.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Teams
+weight: 1
+---
+
+If more than one developer is contributing to a project, it is wiser to set up a team so that everyone could work on the project together. Teams of two are free of charge. It is possible to invite more members by purchasing additional seats. Billing information is available [here](https://codemagic.io/pricing/).
+
+{{<notebox>}}Note that teams who enabled billing before June 6th, 2020, still operate under the previous billing system.{{</notebox>}}
+
+
+## Creating a new team
+
+To create a new team, go to the [app settings](`https://codemagic.io/apps`) and click on the **Teams** icon. Here, click **Create new team**. Choose a suitable name for your team, add the app(s) you want to share by ticking relevant boxes and then click **Create team**. If you don't have any apps to share at this point, you can just add them later.
```

Comment:
> add the app(s) you want to share...
Perhaps it would be better if instead of "share", it was something like "you want/plan to work on"?